### ISSUE TITLE
get the OutOfRangeError

### ISSUE THREAD

**Issue by burness** [OlavHN/fast-neural-style#6]
Hi  @OlavHN  Thanks for this fast neural style, I run your code with the pretrain model and fail with an error 'tensorflow.python.framework.errors.OutOfRangeError'. I run with this command "python fast_neural_style.py --CONTENT_IMAGES ./images" and  in images there are two images  but It can't be successful.

I have print the image in the code 

It seems "content images Tensor("batch:0", shape=(1, ?, ?, 3), dtype=float32) "

Could you help me ? Thanks

**Comment by burness** [comment]
@OlavHN  I solve this problem just remove the num_epochs in the "    filename_queue = tf.train.string_input_producer(filenames, num_epochs=epochs, shuffle=shuffle)
"  or  increase the epochs in ```
content_images = reader.image(
                FLAGS.BATCH_SIZE,
                FLAGS.IMAGE_SIZE,
                FLAGS.CONTENT_IMAGES_PATH,
                epochs=2,
                shuffle=False,
                crop=True)

```
```

**Comment by OlavHN** [comment]
Thanks for reporting.

`inference.py` no longer uses an input queue to make it simpler. Still haven't pushed any pretrained models tho.
